UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -61,7 +61,7 @@ public class CharacterCustomization : MonoBehaviour
 		public CharacterDir currentDir;
 
 		[SerializeField] private List<Color> availableSkinColors;
-		private CharacterSettings currentCharacter;
+		public CharacterSettings currentCharacter;
```

Comment:
do you need to set this from outside or only read it from outside?

Reply:
https://softwareengineering.stackexchange.com/questions/161303/is-it-bad-practice-to-use-public-fields

Reply:
done.